.github/workflows/release.yml

```diff
@@ -0,0 +1,20 @@
+name: Release New Version
+
+on:
+  push:
+    tags:
+      - "v*.*.*"
+
+jobs:
+  build:
+    # change runner when using a public repo. See https://salesforce.quip.com/bu6UA0KImOxJ
```

Comment:
Given public repo, should remove quip.com reference.

Reply:
Removing.
Is there a reason why this repo is public? All of our other repos are private.